task 3.txt

```diff
@@ -0,0 +1,5 @@
+solution :
+-file for managing the features and users (csv or yml)
+-secret manager for storing the password and other needed info of the users
+-creating ps script that install the features and creates the users with their data on the server
+-make ansible playbook that copy the files to the machine and than run the script with the list file.
```

Comment:
What if each environment has different users?

Reply:
I will use naming convension for the secret manager.
In the script I will pass parameter that will describe for which environment is the installation.
I will ensure that the IAM role attached to each environment has the permissions to access the relevant secrets in aws secrets manager.
If I create ci/cd pipelines for this proccess, I will config them to pass the environment as a variable